drupal.make

```diff
@@ -3,10 +3,11 @@ api = 2
 
 ; Core
 projects[drupal][type] = core
-projects[drupal][version] = 7.54
-projects[drupal][patch][] = http://drupal.org/files/issues/menu-get-item-rebuild-1232346-45.patch
-projects[drupal][patch][] = http://drupal.org/files/ssl-socket-transports-1879970-13.patch
-projects[drupal][patch][] = http://drupal.org/files/issues/translate_role_names-2205581-1.patch
+projects[drupal][version] = 7.56
+projects[drupal][patch][] = "http://drupal.org/files/issues/menu-get-item-rebuild-1232346-45.patch"
+projects[drupal][patch][] = "http://drupal.org/files/ssl-socket-transports-1879970-13.patch"
+projects[drupal][patch][] = "http://www.drupal.org/files/issues/1232416-autocomplete-for-drupal7x53.patch"
+projects[drupal][patch][] = "http://drupal.org/files/issues/translate_role_names-2205581-1.patch"
```

Comment:
This does not seem to include all the patches mentioned in the README e.g. https://www.drupal.org/files/issues/DATE_RFC7231-2877243-26.patch seems to be missing.

General comments not related to this PR:
- https would be nice.
- patches should have comments.

Reply:
Patch removed as it's part of the 7.56 release of core.